src/plugins/intel_gpu/src/graph/non_zero.cpp

```diff
@@ -80,7 +108,9 @@ std::string gather_nonzero_inst::to_string(gather_nonzero_node const& node) {
 
     json_composite gather_nonzero_info;
     gather_nonzero_info.add("input id", input.id());
-    gather_nonzero_info.add("output layout", node.get_output_layout().to_string());
+    if (node.is_valid_output_layout()) {
+        gather_nonzero_info.add("output layout", node.get_output_layout().to_string());
```

Comment:
I think saving output layout here is not needed as it's supposed to be saved in the base class (node.desc_to_json())

Reply:
Done

Reply:
Done